packages/now-node/test/fixtures/48-change-handler-by-env/now.json

```diff
@@ -0,0 +1,20 @@
+{
+  "version": 2,
+  "builds": [
+    {
+      "src": "index.js",
+      "use": "@now/node"
+    }
+  ],
+  "build": {
+    "env": {
+      "NODEJS_AWS_HANDLER_NAME": "myCustomHandler"
```

Comment:
```suggestion
      "NODEJS_AWS_HANDLER_NAME": "index.myCustomHandler"
```

Reply:
No @styfle. The build env var is available to all serverless functions, so it only lets you specify the handler name.

Reply:
@styfle this is actually not required, since we always use the entrpoint for the file and just use `handler.split('.').pop()` to get the function.

Reply:
You're right! It used to require a the file name but I realize that was changed so it should work as-is. 👍